hosts/hosts.go

```diff
@@ -420,6 +420,15 @@ func (h *Host) GetExtraArgs(service v3.BaseService) map[string]string {
 	}
 }
 
+func (h *Host) GetExtraArgsArray(service v3.BaseService) map[string][]string {
+	switch {
+	case h.OS() == "windows" && len(service.WindowsExtraArgsArray) > 0:
```

Comment:
If the `ExtraArgsArray` is supposed to be used when the OS is NOT windows, in the case where we have `ExtraArgsArray` but not `WindowsExtraArgsArray`, will it return the `ExtraArgsArray`? 

```suggestion
	case h.OS() == "windows":
```

Reply:
Yes it will, this functions in the same fashion as the function above it `GetExtraArgs`

Reply:
I am ok with it if that is how we handle the same case at all places.